test/clj/game/cards/operations_test.clj

```diff
@@ -140,6 +140,17 @@
       (click-card state :corp "Breaking News")
       (click-prompt state :corp "BOOM!")))
 
+(deftest accelerated-diagnostics-trashes-unplayed-cards
+    ;; No additional costs
+    (do-game
+      (new-game {:corp {:deck ["Accelerated Diagnostics" (qty "Ice Wall" 4)]}})
+      (starting-hand state :corp ["Accelerated Diagnostics"])
```

Comment:
Just put Accelerated Diagnostics into the :hand vector in new-game lol.

Reply:
Whoops, just copy pasted that from an older test. Updated.